eldritch/encounters.py

```diff
@@ -119,11 +119,11 @@ def Roadhouse3(char):
 
 
 def Roadhouse4(char):
-  return events.Nothing()  # TODO buying stuff
+  return events.Purchase(char, "common", 3, keep_count=3, discount_type="fixed", discount=-1)
```

Comment:
Please add tests. For the encounters where a monster appears, nothing fancy is needed - maybe just check that the player gets to a fight or evade choice?